virtcontainers/qemu.go

```diff
@@ -1051,33 +1058,64 @@ func (q *qemu) hotplugMemory(memDev *memoryDevice, op operation) (int, error) {
 		return 0, fmt.Errorf("cannot hotplug negative size (%d) memory", memDev.sizeMB)
 	}
 
-	// We do not support memory hot unplug.
-	if op == removeDevice {
-		// Dont fail for now, until we fix it.
-		// We return that we only unplugged 0
-		q.Logger().Warn("hot-remove VM memory not supported")
-		return 0, nil
-	}
-
+	q.Logger().Debugf("hotplug %dMB mem", memDev.sizeMB)
```

Comment:
Nit: This would be better as:

```suggestion
	q.Logger().WithField("hotplug-memory-mb", memDev.sizeMB).Debug("hotplugging memory")
```